scvi/dataset/dataset.py

```diff
@@ -1237,6 +1238,44 @@ def reorder_cell_types(self, new_order: Union[List[str], np.ndarray]):
     #                           #
     #############################
 
+    def to_anndata(self) -> anndata.AnnData:
+        """
+            Converts the dataset to a anndata.AnnData object.
+            The obtained dataset can then be saved/retrieved using the anndata API.
+        """
+
```

Comment:
what happens to dataset.cell_types here? I think these mappings are in dataset.attribute_mappings?

Reply:
I manually fixed the cell_types situation.

Reply:
I manually fixed the cell_types situation.